client/src/Clusters/TopCharts/TopCharts.js

```diff
@@ -0,0 +1,191 @@
+        title: "App Name",
+        category: "Developer",
+        rating: 4.6
+      },
+      {
+        listNum: 4,
+        url: emptyAppImage,
+        title: "App Name",
+        category: "Developer",
+        rating: 4.6
+      }
+    ];
+ 
+    const topGrossing = [
+      {
+        listNum: 1,
+        url:
+          "https://lh3.googleusercontent.com/MoJxQ-5RA73b3Wffj91crqXUb-C5LKRYCADm11KbfykIkmNbicXGDa3g9X62C_buEg",
+        title: "Pandora",
+        category: "Developer",
+        rating: 4.6
+      },
+      {
+        listNum: 2,
+        url: emptyAppImage,
+        title: "App Name",
+        category: "Developer",
+        rating: 4.6
+      },
+      {
+        listNum: 3,
+        url: emptyAppImage,
+        title: "App Name",
+        category: "Developer",
+        rating: 4.6
+      },
+      {
+        listNum: 4,
+        url: emptyAppImage,
+        title: "App Name",
+        category: "Developer",
+        rating: 4.6
+      }
+    ];
+ 
+    const trending = [
+      {
+        listNum: 1,
+        url:
+          "https://is4-ssl.mzstatic.com/image/thumb/Purple123/v4/2a/24/92/2a249225-7b8d-1ee0-549a-0b09d8f2930d/AppIcon-0-0-1x_U007emarketing-0-0-0-5-0-0-sRGB-0-0-0-GLES2_U002c0-512MB-85-220-0-0.png/246x0w.png",
+        category: "Developer",
+        rating: 4.6
+      },
+      {
+        listNum: 2,
+        url: emptyAppImage,
+        title: "App Name",
+        category: "Developer",
+        rating: 4.6
+      },
+      {
+        listNum: 3,
+        url: emptyAppImage,
+        title: "App Name",
+        category: "Developer",
+        rating: 4.6
+      },
+      {
+        listNum: 4,
+        url: emptyAppImage,
+        title: "App Name",
+        category: "Developer",
+        rating: 4.6
+      }
+    ];
+ 
+    const topPaid = [
+      {
+        listNum: 1,
+        url:
+          "https://is2-ssl.mzstatic.com/image/thumb/Purple123/v4/50/4f/a6/504fa6fe-d52c-6051-8914-d07d78433567/AppIcon-0-0-1x_U007emarketing-0-0-0-7-0-0-sRGB-0-0-0-GLES2_U002c0-512MB-85-220-0-0.png/246x0w.png",
+        title: "MyWeather Radar Weather",
+        category: "Developer",
+        rating: 4.6
+      },
+      {
+        listNum: 2,
+        url: emptyAppImage,
+        title: "App Name",
+        category: "Developer",
+        rating: 4.6
+      },
+      {
+        listNum: 3,
+        url: emptyAppImage,
+        title: "App Name",
+        category: "Developer",
+        rating: 4.6
+      },
+      {
+        listNum: 4,
+        url: emptyAppImage,
+        title: "App Name",
+        category: "Developer",
+        rating: 4.6
+      }
+    ];
+ 
+    return (
+      <div className="col-12 bg-light mx-auto text-center">
+        <div class="sectionHeader">
+          <h4>Top charts</h4>
+          <h8>For Wear OS </h8>
+        </div>
+        <div class="row" id="Top_Charts_Categories">
+          <div class="col-sm-3">
+            <h6 class="chart_categories"> TOP FREE </h6>
```

Comment:
include of hardcoding the text to be all uppercase, you should do this in css as it is a styling decision. https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform